linode/manager
# M3-329 - Add integration test junit logging

* Adds junit logging for  integration tests.
* Needed to record the results on jenkins

## To Test
0. Run selenium
1. Run manager or storybook depending on what you'd like to test

2. Simply add `--log` to any `yarn e2e` test command. 
`yarn e2e --log`
`yarn storybook:e2e --log`

Logs should be generated and saved to `./storybook-test-results`  and `e2e/test-results`
